docs/api_main.md

```diff
@@ -7,17 +7,40 @@ nav_order: 10
 ---
 
 # API
-HTTP requests for getting relevant information from some of our services have been put in a Postman Collection for easy access
+Web requests for getting relevant information from some of our services have been put in a Postman collection
 
-Currently this collection includes requests to the following SmartClean services:
+To access the APIs, please the below link using [Postman](https://www.postman.com) (web or desktop client)
+Note: You need to create a Postman account to sign in (if you are not using one already)
+
+Please find link to the postman collection here:
+[Postman collection](https://www.getpostman.com/collections/a41502dfa9b37c964177)
+
+Currently, this collection includes requests to the following services:
 1. Grids
 2. Device Management
 3. Metrics
 
-This collection can be imported using Postman (web or desktop client)
+## Documentation
 
-Please find link to the postman collection here:
-[Postman collection](https://www.getpostman.com/collections/a41502dfa9b37c964177)
+View documentation for these APIs here: 
+[Documentation](https://documenter.getpostman.com/view/2593073/UVktqZWQ)
+
+## How to access these APIs
+
+You will need to open Postman (desktop client or web console) and sign in to your Postman account first.
```

Comment:
and sign in to your Postman account optionally.

Reply:
Please note, Sign in to Postman account is mandatory (and saying "optionally" will be misleading) without which the Workspace is not loaded, and user cannot view collections.

I tested this on the web by signing out from my account, the API console is only available after signing in.

Reply:
Thank you for pointing out that the sign in is optional for Desktop app.
For web console, sign in is mandatory
Both these facts have been included in the document now.